webapp/src/app.tsx

```diff
@@ -51,25 +54,25 @@ if (Utils.isDesktop() && Utils.isFocalboardPlugin()) {
             return
         }
 
-        history.replace(pathName.replace((window as any).frontendBaseURL, ''))
+        history.replace(pathName.replace(window.frontendBaseURL, ''))
     })
 }
 
 const browserHistory = {
     ...history,
-    push: (path: string, ...args: any[]) => {
+    push: (path: string) => {
```

Comment:
Because we are overriding it, we shouldn't change the API,I think the right approach is to reuse the definition shown here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/history/index.d.ts